pages/static/src/scss/components/_nav.scss

```diff
@@ -561,47 +528,51 @@
     background: darken(#f5f5f5, 10);
   }
 }
-// 
-// @media (min-width: 768px) {
-// 
-//   .navbar-brand {
-//     padding: 15px;
-//   }
-// 
-//   .navbar .logo {
-//     height: 50px;
-//     width: auto;
-//     transition: 0.25s;
-// 
-//     &:hover {
-//       transform: scale(1.1);
-//     }
-//   }
-// 
-//   .navbar {
-//     height: 80px;
-//   }
-// 
-//   .navbar-default .navbar-nav > li > a {
-//     padding-top: 30px;
-//     padding-bottom: 30px;
-//   }
-// 
-//   .navbar .btn-donate.navbar-btn {
-//     margin-top: 17px;
-//     margin-bottom: 17px;
-//     padding: 11px 18px;
-//   }
-// 
-//   .navbar-nav--centered {
-//     text-align: center;
-//     position: absolute;
-//     left: 0;
-//     right: 0;
-// 
-//     > li {
-//       float: none;
-//       display: inline-block;
-//     }
-//   }
-// }
+
+.navbar--microsite {
+  .microsite-header-image {
+    width: auto;
+    height: 150px;
+  }
+
+  .navbar__upper {
+    height: auto;
+  }
+
+  @include until($small) {
+    .navbar__upper {
+      border-bottom: none;
+      display: block;
+    }
+
+    .microsite-header-image {
+      display: block;
+      margin: 50px auto 0;
+    }
+  }
+
+  @include until($medium) {
+    .navbar__upper {
+      border-bottom: none;
+      display: block;
+    }
+
+    .microsite-header-image {
+      display: block;
+      margin: 50px auto 0;
+    }
+
+    .navbar__social {
```

Comment:
Suggest to eventually move these out of top left, maybe make follow up ticket? Or add code comments?

Reply:
I think should have broader conversation since this follows convention of our site nav.

Reply:
Haha I didn't even realize this is same layout as our main site nav 🤦‍♂️

Reply:
That was done intentionally and we could potentially A/B test swapping social and donate icons but need a bigger conversation about it.

Reply:
I did not mean to swap them. I meant more to rethink what goes in top left area since it's first thing user usually sees.